CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Cromwell Change Log
 
+## 60 Release notes
+
+### Warning! Breaking Change: Carboniting Removed
```

Comment:
Note: This part is worth reading even if you only quickly skim over everything else